Xamarin.Forms.Core/IPanGestureController.cs

```diff
@@ -1,6 +1,6 @@
-﻿namespace Xamarin.Forms
+﻿namespace Xamarin.Forms.Internals
 {
-	internal interface IPanGestureController
+	public interface IPanGestureController
```

Comment:
It was moved because it was previously internal. Will move it back to XF namespace and expose publicly. Suggest you review to make sure it's the API you want exposed.